pull_request_template.md

```diff
@@ -12,12 +12,16 @@ Technical details for the knowledge of other developers.
 **Mockup:**
 https://invis.io/RYA3XN5WP#/273832670_Homepage_2-2_E
 
-The following are ALL required for the PR to be merged:
-- [ ] Code review
-- [ ] Scheduled Demo including Design/Testing/Front-end OR Provided Instructions for Local Testing above and in JIRA (if applicable)
-- [ ] Tagged Designer in `#us-ux-frontend` Slack Channel in the thread under the Post from Github for their SA
-- [ ] [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated (if applicable)
-- [ ] [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged (if applicable)
+The following are ALL required for the PR to be merged (in ascending LOE):
+- [ ] Link to this PR in JIRA ticket
+- [ ] Tagged Design, Testing, and Code Reviewers in `#us-ux-frontend` Slack Channel in the thread under the Post from Github for their SA
+- [ ] Scheduled demo including Design/Testing/Front-end `if applicable`
+- [ ] Provided instructions for testing in JIRA (for benefit of testing) and PR (for benefit of code reviewer) `if applicable`
 - [ ] Verified cross-browser compatibility
+- [ ] Verified mobile/tablet/desktop/monitor responsiveness
+- [ ] Added Unit Tests for new Container Components `if applicable`
```

Comment:
I would either remove this (as it is expected as a prerequisite to creating the PR), or if we choose to keep it as a reminder, add that we should also update unit tests for helper functions and reducers.

Reply:
yeah, this should be removed. or in addition to the above it should say added tests for any methods or additional logic in containers.